plugins/woocommerce-beta-tester/src/live-branches/components/BranchList.tsx

```diff
@@ -109,7 +109,7 @@ const BranchInfo = ( { branch }: { branch: Branch } ) => {
 
 const WooCommerceVersionInfo = () => {
 	// @ts-ignore
-	const version = window?.wc?.WC_VERSION || 'unknown';
+	const version = window?.wc?.wcSettings?.WC_VERSION || 'unknown';
```

Comment:
This fixes a bug where it was not determining the WC version properly and showing `unknown` all the time.